### PR TITLE
Improve /help youtube

### PR DESCRIPTION
**Pull request by Distrib-ps**
The Poll and dc have been improved. I did the same for Youtube.

### CODE REVIEW

File: server/chat-plugins/youtube.ts [hunk 1]
```diff
@@ -573,25 +573,27 @@ export const commands: ChatCommands = {
...
-		`/youtube decategorize [channel name] - Removes the category for the [channel], if there is one. Requires: @ # &`,
-		`/youtube categores - View all channels sorted by category.`,
-		`/youtube groupwatch [link], [title] - Creates a group watch of the [url] with the given [title]. Requires % @ & #`,
-		`/youtube startwatch - Starts the group watch in the current room, if there is one. Requires % @ & #`,
-		`/youtube stopwatch - Ends the current group watch, if there is one in the current room. Requires % @ & #`,
-	],
+	youtubehelp() {
+		const help = [
+			`<code>/randchannel [optional category]</code> - View data of a random channel from the YouTube database.` +
+				` If a category is given, the random channel will be in the  given category.`,
```
Comment (AnnikaCodes) [inline server/chat-plugins/youtube.ts:579]:
```suggestion
			`<code>/randchannel [optional category]</code> - View data of a random channel from the YouTube database. If a category is given, the random channel will be in the  given category.`,
```

File: server/chat-plugins/youtube.ts [hunk 2]
```diff
@@ -573,25 +573,27 @@ export const commands: ChatCommands = {
...
+			`<code>/youtube addchannel [channel]</code> - Add channel data to the YouTube database. Requires: % @ #`,
+			`<code>/youtube removechannel [channel]</code>- Delete channel data from the YouTube database. Requires: % @ #</summary>` +
+			`<code>/youtube channel [channel]</code> - View the data of a specified channel. Can be either channel ID or channel name.`,
```
Comment (AnnikaCodes) [inline server/chat-plugins/youtube.ts:582]:
```suggestion
			`<code>/youtube removechannel [channel]</code> - Delete channel data from the YouTube database. Requires: % @ #</summary>`,
			`<code>/youtube channel [channel]</code> - View the data of a specified channel. Can be either channel ID or channel name.`,
```

**Comment by mia-pi-git** [comment]
Is there a gain for this?
Code formatting makes it almost harder to read, as well as the fact this adds more bandwidth and size to the help command when it doesn't need it.
I'll leave this for a bit but personally I would deny this without a better argument.

**Comment by Distrib-ps** [comment]
> Is there a gain for this?
> Code / readmore formatting makes it almost harder to read, as well as the fact this adds more bandwidth and size to the help command when it doesn't need it.
> This isn't a command that gets broadcasted much or ever (only YT staff need it) so there's not much gain to the readmore other than more inconvenience.
> I'll leave this for a bit but personally I would deny this without a better argument.


I just wanted to do like the other help command.
Yes, youtube staff only use it.
So I close this pr.

**Pull request closed without merging**